### DISCUSSION TITLE
Exposure time finder 8k scaling

### DISCUSSION THREAD

**Discussion by beattie** [sn4k3/UVtools#828] [Q&A] [answered]
I've got an 8k 10" printer (HALOT Mage) and the Exposure time finder generate images too small to resolve. I got around most of that by switching units of measure to mm instead of pixels, but the text that give the parameters next to the counter triangles is way too small. Is there some way to scale that other than scaling the whole thing?

**Comment by sn4k3** [comment, marked as answer]
Generate the test with default px measures and after use Tool - Resize.
Still read this whole topic: https://github.com/sn4k3/UVtools/discussions/781#discussioncomment-7775010
